portfolio/src/main/webapp/index.html

```diff
@@ -72,18 +72,17 @@ <h3>Aspiring Software Engineer</h3>
 		<div class="container">
 			<div class="row">
 				<div class="about">
-					<center>
-					<div class="clearfix"></div>
-					<h2>Courses Taken</h2>
+                    <center>
+					<h2>Courses</h2>
 					<p class="struct">
 						Data Structures and Algorithms (CS 225)<br>
 						Computer Architecture (CS 233)<br>
+                        Software Programming Studio (CS 126)<br>
 						Discrete Structures (CS 173)<br>
 						Numerical Methods (CS 357)<br>
-						Intro to Computer Science (CS 125) <br>
 				</div></center>
-				<div class="about">
-					<center><img class="img-responsive" src="images/study.png" height="300" width="300"/></center>
+				<div class="image">
```

Comment:
There is another `id="image"` on line 61, did you mean to use `id`? Or use `class` above? It's a good practice to give unique names if possible, as otherwise it can be confusing to find which `image` to look for in the CSS file.